packages/dds/tree/src/test/simple-tree/api/storedSchema.spec.ts

```diff
@@ -13,35 +13,40 @@ import {
 import { testSimpleTrees } from "../../testTrees.js";
 import { takeJsonSnapshot, useSnapshotDirectory } from "../../snapshots/index.js";
 import { typeboxValidator } from "../../../external-utilities/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { supportedSchemaFormats } from "../../feature-libraries/schema-index/codecUtil.js";
 
 describe("simple-tree storedSchema", () => {
 	describe("test-schema", () => {
 		useSnapshotDirectory("simple-tree-storedSchema");
-		for (const test of testSimpleTrees) {
-			it(test.name, () => {
-				const persisted = extractPersistedSchema(test.schema);
-				takeJsonSnapshot(persisted);
-			});
+		for (const schemaFormat of supportedSchemaFormats) {
+			for (const test of testSimpleTrees) {
+				it(`${test.name} - schema v${schemaFormat}`, () => {
+					const persisted = extractPersistedSchema(test.schema);
+					takeJsonSnapshot(persisted);
+				});
 
-			// comparePersistedSchema is a trivial wrapper around functionality that is tested elsewhere,
-			// but might as will give it a simple smoke test for the various test schema.
-			it(`comparePersistedSchema to self ${test.name}`, () => {
-				const persistedA = extractPersistedSchema(test.schema);
-				const status = comparePersistedSchema(
-					persistedA,
-					test.schema,
-					{
-						jsonValidator: typeboxValidator,
-					},
-					false,
-				);
-				assert.deepEqual(status, {
-					isEquivalent: true,
-					canView: true,
-					canUpgrade: true,
-					canInitialize: false,
+				// comparePersistedSchema is a trivial wrapper around functionality that is tested elsewhere,
+				// but might as will give it a simple smoke test for the various test schema.
+				it(`comparePersistedSchema to self ${test.name} - schema v${schemaFormat}`, () => {
+					const persistedA = extractPersistedSchema(test.schema);
+					const status = comparePersistedSchema(
+						persistedA,
+						test.schema,
+						{
+							jsonValidator: typeboxValidator,
+						},
+						false,
+						1,
```

Comment:
I don't see a reason why it wouldn't work already, right?
```suggestion
						schemaFormat,
```

Reply:
Nice catch. I think I changed that first and added the loop later.